ci-operator/step-registry/optional-operators/cvp-common/scorecard/optional-operators-cvp-common-scorecard-commands.sh

```diff
@@ -34,3 +34,13 @@ operator-sdk scorecard --config "${SCORECARD_CONFIG}" \
                        --kubeconfig "${KUBECONFIG}" \
                        --output json \
                        "${OPERATOR_DIR}" > "${ARTIFACT_DIR}"/scorecard-output-basic.json || true
+
+if [ -d "${OPERATOR_DIR}/tests" ]; then
```

Comment:
Can we make this more precise, like `${OPERATOR_DIR}/tests/scorecard/config.yaml` to ensure that the config is actually there?